README.md

```diff
@@ -76,37 +75,86 @@ let dog = new Dog("Spot");
 dog.speak(); // Spot speaks! Spot says woof!
 ```
 In `Dog`s `constructor`, `super` is used as a `method`. Whereas, `Dog`'s `speak()`
-method used `super` as an `object`. When the `super` keyword is used as method, it
+method used `super` as an `object`. When the `super` keyword is used as a method, it
 calls the parent `class` `Pet` with the parameters passed to `Dog`. This is to ensure
 that `Dog` is an instance of `Pet`. When `super` is used as an `object` that refers to
-a `Pet` instance, it can calls methods of the parent `class` `Pet` explicitly.
+a `Pet` instance, it can call methods of the parent `class` `Pet` explicitly.
 
 ## Explain When `class` Inheritance Should Be Used in JavaScript
 
-JavaScript can be a bit confusing since it is a dynamic language that uses syntactical
+You probably won't use inheritance often, but as your code bases get larger, you are more
+likely to find a need for it. If you're creating a number of objects that have similar
+features, then creating a generic object type to contain all the shared functionality,
+and inheriting those features in more specialized objects can be convenient and useful.
+
+JavaScript can be a bit confusing since it is a dynamic language that uses syntactic
```

Comment:
This could actually be broken out into its own sub section

### JavaScript Inheritance is Not Like Other Languages 

(or something like that I'm not sure)